src/Components/Notifications/PartnerOfferCreatedNotification.tsx

```diff
@@ -0,0 +1,116 @@
+import { Flex, Spacer, Text, Box } from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+import { FC } from "react"
+import { useFragment, graphql } from "react-relay"
+import { PartnerOfferCreatedNotification_notification$key } from "__generated__/PartnerOfferCreatedNotification_notification.graphql"
+import { extractNodes } from "Utils/extractNodes"
+import { ExpiresInTimer } from "Components/Notifications/ExpiresInTimer"
+import { BASE_SAVES_PATH } from "Apps/CollectorProfile/constants"
+import { PartnerOfferArtwork } from "Components/Notifications/PartnerOfferArtwork"
+
+interface PartnerOfferCreatedNotificationProps {
+  notification: PartnerOfferCreatedNotification_notification$key
+}
+
+export const PartnerOfferCreatedNotification: FC<PartnerOfferCreatedNotificationProps> = ({
+  notification,
+}) => {
+  const notificationData = useFragment(
+    PartnerOfferCreatedNotificationFragment,
+    notification
+  )
+
+  const {
+    headline,
+    item,
+    targetHref,
+    offerArtworksConnection,
+  } = notificationData
+
+  if (!item || !offerArtworksConnection || !item.partnerOffer) {
+    return (
+      <Text variant="lg" m={4}>
+        Sorry, something went wrong.
```

Comment:
we should probably make better error handling for all notifications, but in a separate PR

Reply:
Good idea! I've already created a ticket for it ([Jira ticket](https://artsyproduct.atlassian.net/browse/ONYX-725)). Let me know if you have any ideas on how it could look. But maybe this short message is enough because it's hard to say what went wrong? 🤔